Assets/Engine/SeedAssetList.seed

```diff
@@ -66,779 +66,75 @@
 		</Class>
 		<Class name="SeedInfo" field="element" version="2" type="{FACC3682-2ACA-4AA4-B85A-07AD276D18A0}">
```

Comment:
How accurate is this seed list file? Is this fully up to date now with all hardcoded C++ asset loads?

Either way, I'm glad to see this updated. This was the item on my list I was most dreading in getting release builds up and running.

Reply:
I didn't verify anything with C++ asset loads.  I changed an error message in AssetBundler (in this review) to pass along the asset hint in order to print out the asset path when it's missing.  I merely went through and removed all the assets that were logged in AssetBundler as missing, plus some spot checking to make sure the assets were indeed not there.

Reply:
In other words, the files listed in this `.seed` file should all exist.